app/views/pizzerias/index.html.erb

```diff
@@ -1,5 +1,11 @@
 <% @pizzerias.each do |pizzeria| %>
   <div class='pizzeria'>
-    <%= pizzeria.name %>
+    <%= link_to(pizzeria.name, pizzeria_path(pizzeria)) %>
+    <% pizzeria.reviews.sort_by{ |review| review.created_at }.reverse.first(3).each do |review| %>
```

Comment:
I would put this in a model method. Also, it's preferable to use SQL for ordering and limiting rows. Here we're getting every review from the DB but we're only displaying three. It would save bandwidth to only retrieve the latest three from the DB:

```ruby
class Pizzeria
  has_many :reviews

  def latest_reviews
    reviews.order(created_at: :desc).limit(3)
  end
end
```

And then,

```HTML+ERB
<% pizzeria.latest_reviews.each do |review| %>
```